### DISCUSSION TITLE
Backing up a single node in Stateful Set

### DISCUSSION THREAD

**Discussion by kontaras** [vmware-tanzu/velero#6655] [Community support Q&A]
I am trying to back up a database, in which all nodes in a stateful set are replicating each other. All healthy pods are storing the same data, so, ideally, I only want to back up the first healthy pod in my set. I could manually implement a script that checks the health of nodes, labels one as a backup target, and then tells velero to only back up that label, but I wonder if anyone has already done something similar and built it into the project (or created a plug-in for it).

**Comment by draghuram** [comment]
I am a bit confused by "back up the first healthy pod". If your intention is to backup replicated data only once, do you mean that you want to backup only one PVC? I am trying to understand whether you are referring to data backup or to the Pod resource itself.

**Reply by kontaras** [reply]
I only need to back up one healthy pod in my stateful set. I can create the other pods in my set using a helm chart to initialize them with blank state and tell them to replicate the restored pod.

**Reply by draghuram** [reply]
Ok. In that case, your solution sounds fine to me. I don't think Velero supports this use case out of the box.
